git/essentials/features-i/get-the-most-out-of-git.md

```diff
@@ -9,7 +9,11 @@ type: normal
 
 category: feature
 
-
+aspects:
+  
+  - deep
```

Comment:
This doesn't really fit with `obscura`, you'll do all of these things regularly; I would keep it `introduction` and `workout` since it has a few tidbits of useful introduction level information